login/login.c

```diff
@@ -308,6 +308,7 @@ int login_prompt(glome_login_config_t* config, pam_handle_t* pamh,
 static char* create_login_message(glome_login_config_t* config,
                                   pam_handle_t* pamh, const char** error_tag) {
   char* host_id = NULL;
+  int max_hostname_len = sysconf(_SC_HOST_NAME_MAX) + 1;
```

Comment:
Should we check return value of sysconf?

Reply:
Fair enough, yes. Let's pick _POSIX_HOST_NAME_MAX (given that it's per standard the minimum, even if Linux disagrees) as the fallback value.